docs/reference/commands.md

```diff
@@ -1988,7 +1987,7 @@ Copy or merge table
 #### Synopsis
 {:.no_toc}
 
-copy or merge table. the destination table will point to the selected branch
+Copy or merge table. the destination table will point to the selected branch
```

Comment:
Is this change related to this pr?

Reply:
No, this was generated with make gen-doc. I believe sone changes from another PR was generated, and since it wasn't included in the relevant PR Barak and I thought it would be Ok to include them here.